store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9569,5 +9569,19 @@ TODO: delete them permanently from here
   <desc>Mark messages sent to a forwarding address as read</desc>
 </attr>
 
+<attr id="2124" name="zimbraFeatureContactBackupFrequency" type="duration" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited,domainAdminModifiable" callback="ContactBackupFeature" since="8.8.5">
```

Comment:
why do these new attributes have "domainAdminModifiable" flag? I don't think ANY new attributes should have this flag as it is used only for legacy domain admin, which has been deprecated since 7.x

Reply:
Raised https://github.com/Zimbra/zm-mailbox/pull/362 to remove domainAdminModifiable flag